source/framework/analysis/inc/TRestEventRateAnalysisProcess.h

```diff
@@ -52,6 +55,11 @@ class TRestEventRateAnalysisProcess : public TRestEventProcess {
     void PrintMetadata() {
         BeginPrintProcess();
 
+        if (fRateAnalysis)
+			   metadata << "Rate analysis is on under single thread mode" << endl;
+		  else
+			   metadata << "Rate analysis is off due to multi-thread" << endl;
```

Comment:
I'd avoid if expressions without curly brackets. Too easy to introduce bugs if the body is ever extended, especially because indentation (which is mixing tabs & spaces here btw) doesn't save you in C/C++.